src/Command/Ssh/SshKeyListCommand.php

```diff
@@ -33,21 +35,57 @@ protected function execute(InputInterface $input, OutputInterface $output) {
     $local_keys = $this->findLocalSshKeys();
 
     $table = new Table($output);
-    $table->setHeaders(['Local Key Filename', 'Cloud Platform Key Label']);
+    $table->setHeaders([
+      'Cloud platform key label and UUID',
```

Comment:
`platform` should take a capital `P`

Also, I don't have a better suggestion, but I don't really like the label and UUID side-by-side. It lacks readability.

Reply:
Yeah I'm wondering if we should change the display so that it prints sections rather than a table. E.g.,

```
-------------------
Label:
UUID:
Filename:
sha256:
md5:
-------------------
Label:
UUID:
Filename:
sha256:
md5:
-------------------
```

Reply:
Just did some prototyping, would be:

```
Cloud keys with matching local keys
===================================

 ---------------- -------------------------------------------------
  Label            id_rsa_ac
  UUID             ...
  Local filename   id_rsa.pub
  sha256 hash      ...
  md5              ...
 ---------------- -------------------------------------------------

 ---------------- -------------------------------------------------
  Label            MattGrasmickMBP
  UUID             ...
  Local filename   acquia_cloud.pub
  sha256 hash      ...
  md5              ...
 ---------------- -------------------------------------------------

Cloud keys with no matching local keys
======================================

 ---------------- -------------------------------------------------
  Label            PT Travis CI
  UUID             ...
  Local filename   none
  sha256 hash      ...
  md5              ...
 ---------------- -------------------------------------------------
```

Reply:
@anavarre what do you think about this output format?

Reply:
I was more in favor of keeping the current command simple and having an `ssh-key:info` command specifically. If you want to keep it all with one command, then I guess that works.